### DISCUSSION TITLE
Adding Beatmap to Collections should not close context menu

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#25047] [Ideas] [answered]
This should be a huge quality of life feature for people who has many collections and wants to add to multiple collections in one go

You can see how annoying this can be:
![osu!_6YT3uYeZkG](https://github.com/ppy/osu/assets/64581009/a986b55b-f7eb-44fa-8ee6-610d921d13f5)

**Comment by Joehuu** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/10255, can add via `All beatmaps` dropdown without it closing for now.
